.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+*  @guardian/dotcom-platform @guardian/mss-admins
```

Comment:
The github reports this error: ```Unknown owner on line 1: make sure the team @guardian/mss-admins exists, is publicly visible, and has write access to the repository```.  Do we need to deal with it?

Reply:
The team name `@guardian/mss-admins` is correct - https://github.com/guardian/mobile-apps-api/blob/main/.github/CODEOWNERS

Reply:
👍 the team didn't have write access, @groakland and I have updated this.